packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-load-balancer.ts

```diff
@@ -51,6 +51,14 @@ export interface ApplicationLoadBalancerProps extends BaseLoadBalancerProps {
    * @default false
    */
   readonly dropInvalidHeaderFields?: boolean;
+
+  /**
+   * Determines how the load balancer handles requests that
+   * might pose a security risk to your application
+   *
+   * @default DesyncMitigationMode.DEFENSIVE
```

Comment:
```suggestion
   * @default - desync mitigation mode is not enabled
```

Right?

Reply:
Thank you for the review!
Without `routing.http.desync_mitigation_mode` attribute set, the default is actually defensive mode on the resource level.

https://docs.aws.amazon.com/elasticloadbalancing/latest/application/application-load-balancers.html

> routing.http.desync_mitigation_mode
> Determines how the load balancer handles requests that might pose a security risk to your application. The possible values are monitor, defensive, and strictest. The default is defensive.

Reply:
ahh ok!

Reply:
Sure. no worries